pkg/api/clustermanager.go

```diff
@@ -0,0 +1,153 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// SyncSetList represents a list of SyncSets for a given cluster.
+type SyncSetList struct {
+	SyncSets []*SyncSet `json:"value"`
+	// The link used to get the next page of operations.
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+type ClusterManagerConfigurationList struct {
+	ClusterManagerConfigurations []*ClusterManagerConfiguration `json:"value"`
+
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+// ClusterManagerConfiguration represents the configuration from OpenShift Cluster Manager (OCM)
+type ClusterManagerConfiguration struct {
+	MissingFields
```

Comment:
technically this is bad according to resource provider contract, but we're doing this in the ocp doc so :shrug: 

We should address it in the future for all instances.

Reply:
https://eng.ms/docs/products/arm/api_contracts/guidelines/openapi

OAPI018: Reject unknown properties and property values

Reply:
would leave the MissingFields key in the documents, but not in the "API" definitions (internal or external)